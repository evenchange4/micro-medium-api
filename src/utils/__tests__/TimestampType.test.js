const TimestampType = require('../TimestampType');

it('should return TimestampType', () => {
  expect(TimestampType).toMatchSnapshot();
});

it('should support parseValue', () => {
  expect(TimestampType.parseValue(1502421324185)).toBe(1502421324185);
});

it('should support serialize', () => {
  expect(TimestampType.serialize(1502421324185)).toBe(1502421324185);
});
