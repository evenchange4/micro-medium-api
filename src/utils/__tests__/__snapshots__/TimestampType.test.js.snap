// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return TimestampType 1`] = `"Timestamp"`;
